docs/configuration/hyperparameter_optimization.md

```diff
@@ -8,8 +8,8 @@ hyperopt:
   metric: loss
   split: validation
   parameters:
-    text.cell_type: ...
-    text.num_layers: ...
+    title.cell_type: ...
```

Comment:
Is there any way to include an inline comment to completely erase ambiguity?
```
title.cell_type: ... // a text feature type
```

Reply:
Done - this actually does get mentioned further down in the documentation